streaming/src/main/scala/org/apache/spark/streaming/ui/StreamingPage.scala

```diff
@@ -241,24 +241,24 @@ private[ui] class StreamingPage(parent: StreamingTab)
 
     // Use the max input rate for all InputDStreams' graphs to make the Y axis ranges same.
     // If it's not an integral number, just use its ceil integral number.
-    val maxEventRate = eventRateForAllStreams.max.map(_.ceil.toLong).getOrElse(0L)
-    val minEventRate = 0L
+    val maxRecordRate = recordRateForAllStreams.max.map(_.ceil.toLong).getOrElse(0L)
+    val minRecordRate = 0L
 
     val batchInterval = UIUtils.convertToTimeUnit(listener.batchDuration, normalizedUnit)
 
     val jsCollector = new JsCollector
 
-    val graphUIDataForEventRateOfAllStreams =
+    val graphUIDataForRecordRateOfAllStreams =
       new GraphUIData(
-        "all-stream-events-timeline",
-        "all-stream-events-histogram",
-        eventRateForAllStreams.data,
+        "all-stream-records-timeline",
```

Comment:
@srowen thank you for the detailed review! 

Yeah normally this requires something else to change, but `GraphUIData` has been implemented well enough to save the trouble. We only need to change this `"all-stream-events-timeline"` here, then the div ID as well as any other place referring to this div will change accordingly; please see the snapshots of the HTML source:

div as a placeholder:
![records-2-html-1](https://cloud.githubusercontent.com/assets/15843379/14131528/bae892e6-f66e-11e5-9d70-301d06558ff7.png)

then fill in that div with data:
![records-2-html-2](https://cloud.githubusercontent.com/assets/15843379/14131529/bb2bd4b6-f66e-11e5-9386-a8fae9d88d48.png)

Thanks!